chrome-helper/eos-google-chrome.py

```diff
@@ -107,12 +107,27 @@ def _get_chrome_flatpak_launcher(self):
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', dest='debug', action='store_true')
+    parser.add_argument('--ozone-platform', dest='ozone_platform')
+    parser.add_argument('--ozone-platform-hint', dest='ozone_platform_hint')
 
     parsed_args, otherargs = parser.parse_known_args()
 
     if parsed_args.debug:
         logging.root.setLevel(logging.DEBUG)
 
+    ozn_plt = ''
+    if parsed_args.ozone_platform:
+        ozn_plt = f'--ozone-platform={parsed_args.ozone_platform}'
+        otherargs.append(ozn_plt)
+
+    if parsed_args.ozone_platform_hint:
+        ozn_plt_hint = f'--ozone-platform-hint={parsed_args.ozone_platform_hint}'
+        otherargs.append(ozn_plt_hint)
+    elif ozn_plt == '':
+        # Make Chrome use the current XDG session type automatically
+        ozn_plt_hint = f'--ozone-platform-hint=auto'
+        otherargs.append(ozn_plt_hint)
```

Comment:
I think this can be simplified by using `parsed_args` more directly:

```
# If an explicit platform has been provided, use that. Otherwise pass the platform hint.
if parsed_args.ozone_platform:
    otherargs.append(f'--ozone-platform={parsed_args.ozone_platform}')
else:
    otherargs.append(f'--ozone-platform-hint={parsed_args.ozone_platform_hint}')
```

Combined with setting the `ozone_platform_hint` default to `auto`, it should DTRT.

Reply:
Thanks!  That is simpler!